src/Microsoft.AspNetCore.SignalR.Client.TS/Microsoft.AspNetCore.SignalR.Client.TS.csproj

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I started with .proj but VS refused to add it to the solution. Hence changing to project to pretend it's a .csproj